FrxReport.inc

```diff
@@ -736,6 +744,13 @@ class FrxReport {
   public function parametersForm($variables = array()) {
     $parms = $this->parametersArray();
     $form =  drupal_get_form('forena_parameter_form', $parms, $variables);
+    // prefix the form with the report description, if applicable
+    if ($this->description && isset($form['params']) && is_array($form['params'])) {
+      $form['params'] = array('forena-report-description' => array(
```

Comment:
@bgilling, I'm just stuffing a new (hardcoded by name) element in the top of the form here.  Is there a better way to do this?